src/packages/documents/documents/workspace/views/info/document-workspace-view-info-history.element.ts

```diff
@@ -96,7 +109,7 @@ export class UmbDocumentWorkspaceViewInfoHistoryElement extends UmbLitElement {
 						label=${this.localize.term('actions_rollback')}
 						look="secondary"
 						slot="actions"
-						@click=${() => alert('TODO: Rollback Modal')}>
+						@click=${this.#onRollbackModalOpen}>
 						<uui-icon name="icon-undo"></uui-icon>
 						<umb-localize key="actions_rollback"></umb-localize>
```

Comment:
This is already set as a label so we don't need to set it in the default slot as well.